articles/references/templates_decorators/registrations.md

```diff
@@ -46,4 +49,24 @@ Here is an example of a template registration.
         }
     }
 
-```
\ No newline at end of file
+```
+
+# Decorator Registration
+
+The same can be said for Decorators as for Templates. Each Decorator class has to have a corresponding Registration class to tell Intent Architect that this Decorator exists and how to create it.
+
+Here is an example of the Registration class:
+
+```csharp
+public class JwtAuthOwinStartupDecoratorRegistration : DecoratorRegistration<IOwinStartupDecorator>
+{
+    public override string DecoratorId => JwtAuthOwinStartupDecorator.Identifier;
+
+    public override IOwinStartupDecorator CreateDecoratorInstance(IApplication application)
+    {
+        return new JwtAuthOwinStartupDecorator(application.SolutionEventDispatcher);
+    }
+}
+```
+
+It has to inherit from `DecoratorRegistration<>` and have the "blueprint" (contract for enterprise developers) class/interface type specified as a generic type parameter.
```

Comment:
Ultimately this should link to our API documentation. We should create a Github issue to get our SDK XML Doc stuff into here and then this would link to it.

Reply:
Indeed. Some of the Core/SDK components could link in well with the code documentation